### ISSUE TITLE
Mismatched output in Jupyter notebooks

### ISSUE THREAD

**Issue by canyon289** [stan-dev/pystan2#530]
#### Summary:
In Jupyter notebooks the warnings appear in the notebook, and the more pertinent model run information appears in the terminal. Is it possible to switch the two?

Normally I would be this picky, however in this case the notebook will be part of ArviZ's extended resources and presentation is just as important as running code.

Also if I need to create a more minimal example please let me know!

#### Description:
Pystan warnings appear in the notebook. The "standard" model outputs in the terminal. It seems like these outputs should be reversed.

##### Terminal Output of model
https://github.com/arviz-devs/arviz_resources/raw/71deca645c2cb7398fc56ccb5abb32ae7961b297/diagnostics/PystanOutputInTerminal.png

#### Reproducible Steps:
Example notebook can be found here
https://github.com/arviz-devs/arviz_resources/blob/n_effective/diagnostics/EffectiveSampleSize.ipynb

#### Current Output:
Warnings are in notebook
"Nominal" output in terminal, and not saved in Jupyter Notebook

#### Expected Output:
"Nominal" output in Jupyter notebook. Warnings too if that's possible, would be nice if its also possible to turn off warnings as an option

#### PyStan Version:
'2.18.0.0'

#### Python Version:
Python 3.6.7

#### Operating System:
Apple OSX

**Comment by canyon289** [comment]
Thanks for this response. When you say turn off all messages is there a specific step you recommend? I appreciate any direction given that I have none myself right now :)

**Comment by ahartikainen** [comment]
This might be something that osx does (osx compilers).

I don't see any warning messages (Windows) on notebook, only on the terminal output.

**Comment by canyon289** [comment]
Good to know! I'll try this on a different OS and see what happens. Thanks Ari!

**Comment by canyon289** [comment]
Got it. Please feel free to treat this as a low (or non priority) issue. I have enough to go off of that I can experiment to see if I can change the behavior. If I end up with new information that leads to new questions do you mind if I post them here?

**Comment by ahartikainen** [comment]
It's same (sampling statements), but many of us use jupyter which swallows those to terminal or use 

    python script.py > output.log

I don't remember how things are on ipython, but I don't use interactive environments (excluding Jupyter) that often to sample with Stan.

**Comment by mohdluqman** [comment]
@ahartikainen @canyon289 can u give me installation instructions of pystan on jupyter notebook with Anaconda (Windows 10). I tried the steps mentioned in [https://pystan.readthedocs.io/en/latest/windows.html](url) i.e. created a new environment with all the mentioned dependencies installed. Then I selected the new environment in anaconda but when I try to fire up notebook it fails with error message **_Multiple Error encountered_**. I have tried various steps using pip and conda both but I am not able to use pystan with even any IDE that is supported in anaconda.
